live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -78,6 +78,29 @@
     path: /etc/issue
     state: absent
 
+#
+# Override the nfs-mountd service execution parameters. By default, the
+# service provides the --manage-gids flag so NFS will be aware of group
+# membership for users in more than 16 groups. However, this flag has the
+# undesirable side-effect of preventing users from accessing files owned by
+# a group they are a member of, unless it's their primary group. As we do not
+# have a great deal of control of our users' groups configuration, we have
+# decided to disable the feature. Since Illumos users were already limited
+# to 16 groups for NFS, this is not a change in behavior.
+#
+- file:
+    path: /etc/systemd/system/nfs-mountd.service.d
+    state: directory
+    mode: 0755
+
+- copy:
+    dest: /etc/systemd/system/nfs-mountd.service.d/args.conf
```

Comment:
I don't feel strongly, but we've used the filename of `override.conf` elsewhere (e.g. the `delphix-mgmt` service). I'd prefer to be consistent, but it doesn't really matter.

Reply:
Sure - I'm a fan of consistency